web/src/settings_org.js

```diff
@@ -1,11 +1,13 @@
 import $ from "jquery";
 
 import pygments_data from "../generated/pygments_data.json";
+import render_compose_banner from "../templates/compose_banner/compose_banner.hbs";
```

Comment:
It would be nice to rename this to be more generic than `compose`

Reply:
I guess we renamed it to `main_view_banner` at some point, which is not exactly helpful:

```
$ cat web/templates/compose_banner/compose_banner.hbs 
<div
  class="main-view-banner {{banner_type}} {{classname}}"
  {{#if user_id}}data-user-id="{{user_id}}"{{/if}}
  {{#if stream_id}}data-stream-id="{{stream_id}}"{{/if}}
  {{#if topic_name}}data-topic-name="{{topic_name}}"{{/if}}>
    {{#if banner_text}}
    <p class="banner_content">{{banner_text}}</p>
    {{else}}
    <div class="banner_content">{{> @partial-block}}</div>
    {{/if}}
    {{#if button_text}}
    <button class="main-view-banner-action-button{{#if hide_close_button}} right_edge{{/if}}" {{#if scheduling_message}}data-validation-trigger="schedule"{{/if}}>{{button_text}}</button>
    {{/if}}
    {{#if hide_close_button}}
    {{!-- hide_close_button is null by default, and false if explicitly set as false. --}}
    {{else}}
    <a role="button" class="zulip-icon zulip-icon-close main-view-banner-close-button"></a>
    {{/if}}
</div>
```

I'm pretty sure we have a bunch of settings banners involving the `alert` class across the settings code base; perhaps worth a conversation in #design about the settings banners, including @sahil839 in the discussion. I'm not at all convinced the right plan is to use the same component here vs. making a different component for settings banners.

(We should probably just be OK merging this as-is in the meantime).